src/components/layout/inputs/DurationInput/DurationInput.jsx

```diff
@@ -0,0 +1,113 @@
+import PropTypes from 'prop-types'
+import React, { useCallback, useState } from 'react'
+
+import InputError from '../Errors/InputError'
+
+const translateMinutesToHours = durationInMinutes => {
+  if (durationInMinutes === null) return ''
+  const hours = Math.floor(durationInMinutes / 60)
+  const minutes = durationInMinutes % 60
+  return `${hours}:${minutes}`
+}
+
+const validHoursDuration = durationInHours => durationInHours.match(/[0-9]*:[0-5][0-9]/)
```

Comment:
```suggestion
const isHoursDurationValid = durationInHours => RegExp(/[0-9]*:[0-5][0-9]/).test(durationInHours)
```
On est intéressé par un true ou false, je trouve ça plus clair dans ce cas là d'appeler une méthode qui nous renvoie directement ce que l'on souhaite.

Reply:
Aaaaah, merci !
J'avais pas trouvé dans les string